main.go

```diff
@@ -4,16 +4,18 @@ import (
 	"beelzebub/builder"
 	"beelzebub/parser"
 	"flag"
-	"fmt"
 
 	log "github.com/sirupsen/logrus"
 )
 
-var quit = make(chan struct{})
-
 func main() {
-	var configurationsCorePath string
-	var configurationsServicesDirectory string
+	// removed unnecessary fmt.Sprintf call
```

Comment:
I really appreciate your work ❤️  I will approve it once you remove all comments